### ISSUE TITLE
Fair number of community registry packages aren't working right now.

### ISSUE THREAD

**Issue by donmccurdy** [aframevr/aframe-registry#59]
I just tested a couple from the A-Frame homepage:

- leap-hands
- audio-analyzer
- firebase
- firebase-broadcast
- json-model
- mountain

That `leap-hands` is broken is my own fault; I ignored the `dist` folder in `.npmignore`. The others seem to have various issues, some missing on NPM, some throwing JavaScript errors.

Not sure how feasible it would be to test entries in an automated way.

**Comment by ngokevin** [comment]
Thanks. Hmm, how did you test them, via Inspector?

Not sure curation can be automated, but it's a lot of work.

**Comment by ngokevin** [comment]
Hmm maybe in the future, a way of reporting broken components on the Registry

**Comment by donmccurdy** [comment]
Yeah, I just opened the inspector and tried to add those components. I think we could automate a test ensuring that the dist file exists (which was the problem for at least two of these), but the JS errors seem harder to catch.

**Comment by ngokevin** [comment]
OK, ensuring the dist sounds easy at least. I'm not sure if through the Inspector is a common way to consume components anymore, so some of them might not be set up to work on Inspector.

**Comment by donmccurdy** [comment]
What do you mean by 'set up to work on inspector'? I guess some of mine do export something like:

```js
AFRAME.register('foo', {...});
```
.... which assumes that AFRAME is in the global scope, and would break for webpacked sites.

**Comment by ngokevin** [comment]
For example `firebase` components aren't really Inspector friendly, not tested there. I do have the `inspector: false` option, but that's not listened to by Inspector yet.

**Comment by donmccurdy** [comment]
Oh I see, makes sense.

**Comment by fernandojsg** [comment]
It seems `alongpath` doesn't work either https://github.com/aframevr/aframe-inspector/issues/464
